mysql-test/include/galera_wait_sync_point.inc

```diff
@@ -0,0 +1,5 @@
+let $wait_timeout = 10;
+SET SESSION wsrep_on = 0;
```

Comment:
Why is wsrep_on set to zero here? Is there some interaction with the select from the information_schema query?

It may be better to have wsrep_sync_wait set to zero and then restored to its prior value.